pulp_ansible/app/galaxy/v3/pagination.py

```diff
@@ -73,27 +73,33 @@ def get_paginated_response_schema(self, schema):
             "type": "object",
             "properties": {
                 "meta": {
-                    "count": {
-                        "type": "integer",
-                        "example": 123,
+                    "type": "object",
```

Comment:
@daviddavis customized pagination for v3 had an issue with the schema, it was not following OpenAPI standard

Reply:
So like `Fixed v3 schema pagination to match OpenAPI standard` would be great.